Makefile

```diff
@@ -64,7 +64,7 @@ check:
 	docformatter --check $(DOCFORMATTER_OPTIONS) $(DOCFORMATTER_FILES)
 	black --check $(BLACK_FILES)
 	blackdoc --check $(BLACKDOC_OPTIONS) $(BLACK_FILES)
-	flake8 $(FLAKE8_FILES)
+	FLAKEHEAVEN_CACHE_TIMEOUT=0 flakeheaven lint $(FLAKEHEAVEN_FILES)
```

Comment:
The `FLAKEHEAVEN_CACHE_TIMEOUT` cache invalidation flag might not be needed once https://github.com/flakeheaven/flakeheaven/pull/87 is merged into `flakeheaven` and a new release is made.